src/test/java/seedu/address/logic/commands/ListPersonCommandTest.java

```diff
@@ -27,15 +29,19 @@ public void setUp() {
     }
 
     @Test
-    public void execute_listIsNotFiltered_showsSameList() {
-        assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
+    public void execute_moduleListIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new ListPersonCommand(), model, ListPersonCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
     @Test
-    public void execute_listIsFiltered_showsEverything() {
+    public void execute_moduleListIsFiltered_showsEverything() {
```

Comment:
```suggestion
    public void execute_personListIsFiltered_showsEverything() {
```
This is not module list being filtered?

Same for lines 32-33 (new index)

Reply:
Yep noticed that as well! Thanks for pointing it out 👍